test/E2ETests/Common/DeploymentUtility.cs

```diff
@@ -201,7 +201,7 @@ private static Process StartMonoHost(StartParameters startParameters, ILogger lo
 
             var hostProcess = Process.Start(startInfo);
             logger.WriteInformation("Started {0}. Process Id : {1}", hostProcess.MainModule.FileName, hostProcess.Id);
-            Thread.Sleep(15 * 1000);
+            Thread.Sleep(25 * 1000);
```

Comment:
Is there no way to "know" whether the target process is done doing its thing? Can the target process somehow notify this process that it's done? Maybe drop a file somewhere as a signal? Or something?